assets/forms/camera_settings.ui

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>CameraSettings</class>
+ <widget class="QWidget" name="CameraSettings">
+  <property name="geometry">
+   <rect>

```

Comment:
this gets explicitly defined? How would this work for say different sized screens and stuff?

Reply:
The dimensions here are irrelevant as it is automatically sized (and resized). It may be possible to remove this from the file but note that the file was automatically generated from the WYSIWYG editor.

Reply:
It seems that the function of this is strictly how to show it in the editor. I tried removing it, changing it to 0, and it just snaps it back to a default.